build.gradle

```
@@ -329,6 +329,18 @@ testClusters.integTest {
         setting("plugins.security.authcz.admin_dn", "\n- CN=kirk,OU=client,O=client,L=test, C=de")
         setting('plugins.security.restapi.roles_enabled', '["all_access", "security_rest_api_access"]')
         setting('plugins.security.system_indices.enabled', "true")
+        setting('plugins.security.system_indices.indices', '[' +
+                '".plugins-ml-config", ' +
+                '".plugins-ml-connector", ' +
+                '".plugins-ml-model-group", ' +
+                '".plugins-ml-model", ".plugins-ml-task", ' +
```

```suggestion
                '".plugins-ml-model",  ' +
                ".plugins-ml-task",
```


or if we are using all the system indices of ml-commons here we can do `".plugins-ml-*`. Similarly for flow framework indices.

---

I can make the suggested change, but I'd rather not use `.plugins-ml-*` since we're trying to emulate the actual configuration of the demo installation config